src/clang-cpp-frontend/clang_cpp_language.cpp

```diff
@@ -36,6 +38,9 @@ void clang_cpp_languaget::force_file_type()
   std::string clangstd = cppstd.empty() ? "-std=c++03" : "-std=c++" + cppstd;
   compiler_args.push_back(clangstd);
 
+  if(!config.options.get_bool_option("no-optimized-cpp-includes"))
+    compiler_args.push_back("-I" + esbmc_cpp_includes());
```

Comment:
`-I` is not the right option, it should be `-isystem`. See also the discussion in <https://github.com/esbmc/esbmc/pull/834#discussion_r937853269>.

Reply:
Done